### ISSUE TITLE
v4 announcements were on old forum which is now dead.

### ISSUE THREAD

**Issue by Pyrolistical** [RiotGames/developer-relations#34]
For example, we cannot read this https://discussion.developer.riotgames.com/articles/6579/player-universally-unique-identifiers-and-a-new-se.html which is critical for those moving from v3 to v4

**Comment by Pyrolistical** [comment]
Found the link https://www.riotgames.com/en/DevRel/player-universally-unique-identifiers-and-a-new-security-layer

The old url should redirect to new one

**Comment by Pyrolistical** [comment]
Another example is https://discussion.developer.riotgames.com/questions/4269/verification-system.html which is linked from https://developer.riotgames.com/api-methods/#third-party-code-v3

**Comment by Pyrolistical** [comment]
Also on discord: 
```Botty McBotFace: Hey there! I'm Botty, let me be the first to welcome you to the Riot API Discord :)

Here's some tips to help you on your way:
1. There's a good chance the AnswerHub already has an answer for you. Be sure to search there, it will help you much faster. https://discussion.developer.riotgames.com/
```

**Comment by Derpthemeus** [comment]
Botty has been updated (https://github.com/Querijn/BottyMcBotface/commit/3bf77c9c3c729d4c3d00649e51385ccbf70ab3fc) and will be redeployed later

**Comment by Querijn** [comment]
Botty has been redeployed.

**Comment by RiotTuxedo** [comment]
Twitter and Discord are the most reliable places to hear about any recent announcements. Both these locations have been updated with the correct link.

The announcements themselves can be found here:
https://www.riotgames.com/en/DevRel

I've posted a banner on the Developer Portal and each user will receive an update upon login.  I've also added/updated the notes on the v3 APIs on the Reference page.

At this time, the old urls don't redirect to the new ones, however with the updates listed above I believe the deprecation announcement is easily discoverable. With that I'm closing out this issue as resolved. Let  me know if you have additional concerns.
